### ISSUE TITLE
[Triage] Pending tasks not completing

### ISSUE THREAD

**Issue by jl94x4** [kieraneglin/pinchflat#246]
I asked Pinchflat to refresh indexer to grab a new ep of a podcast on youtube, but it hasn't grabbed it. Been waiting all day for it to complete but it never does, attached logs below.

The Overlap, is the affected source.

[pinchflat.log](https://github.com/kieraneglin/pinchflat/files/15326792/pinchflat.log)

**Comment by kieraneglin** [comment]
Hey there, thanks for the report!

It looks like the source you added was a video that's _part_ of a playlist rather than the playlist itself. You added 

```
https://www.youtube.com/watch?v=bG56SYP0awk&list=PLfikCHtGjo4l2AJbpkgkmFOxgIQBc9Pn2&pp=iAQB
```

but try deleting that source and trying again with 

```
https://www.youtube.com/playlist?list=PLfikCHtGjo4l2AJbpkgkmFOxgIQBc9Pn2
```

**Comment by jl94x4** [comment]
With the link you provided, I get this issue

![image](https://github.com/kieraneglin/pinchflat/assets/18123855/73534762-a94a-43ac-959d-46071d506438)

**Comment by kieraneglin** [comment]
I think that's related to #236. That's an issue on the `yt-dlp` side of things, but I'm working with them right now to see about getting around that! Sorry for the trouble

**Comment by kieraneglin** [comment]
@jl94x4 thanks for the patience - I found a workaround I'm happy with so this issue will be fixed in `v2024.5.16` which will be live in ~30 mins

**Comment by jl94x4** [comment]
Yep, looking good now!
